Lib/test/test_asyncio/test_locks.py

```diff
@@ -305,6 +305,12 @@ def test_context_manager(self):
 
         self.loop.run_until_complete(f())
 
+    def test_lock_loop_not_running(self):
+        loop = asyncio.new_event_loop()
+        loop.stop()
+        with self.assertWarns(DeprecationWarning):
+            q = asyncio.Lock(loop=loop)
```

Comment:
This can simply be:
```suggestion
            asyncio.Lock()
```

Reply:
Hi I'm not sure if we must remove the `loop=loop`. we need send a loop isn't?

Reply:
Since `asyncio.Lock()` is using `asyncio.get_event_loop()` when the loop isn't passed, we don't need to explicitly create and pass the event loop to `asyncio.Lock()`.

Reply:
I'd also rather not include passing an explicit loop argument in the new tests we're adding, since that argument will be removed entirely in 3.10. We can change the existing tests later when the argument is removed, but any new tests should preferably not include it.

Reply:
There's also one other concern: if explicitly pass the *loop* argument, it's likely the one triggering the deprecation warning rather than creating the asyncio object outside of a coroutine/running event loop. This can potentially be fixed with using `assertWarnsRegex()` (to check for a specific deprecation warning, rather than any deprecation warning), but I think we're better off implementing it in a way that doesn't require a change in 3.10.